compiler/qsc_rir/src/passes/reindex_qubits.rs

```diff
@@ -157,15 +273,20 @@ fn add_mz(program: &mut Program) -> CallableId {
             .expect("should be at least one callable")
             + 1,
     );
-    program.callables.insert(
-        mz_id,
-        Callable {
-            name: "__quantum__qis__mz__body".to_string(),
-            input_type: vec![Ty::Qubit, Ty::Result],
-            output_type: None,
-            body: None,
-            call_type: CallableType::Measurement,
-        },
-    );
+    program.callables.insert(mz_id, builder::mz_decl());
     mz_id
 }
+
+fn add_cx(program: &mut Program) -> CallableId {
+    let cx_id = CallableId(
+        program
+            .callables
+            .iter()
+            .map(|(id, _)| id.0)
+            .max()
+            .expect("should be at least one callable")
+            + 1,
+    );
```

Comment:
Just noting that I spent about 10 minutes poking around thinking "there's got to be a better way".... but... no, there isn't.